.github/workflows/package.yml

```
@@ -0,0 +1,30 @@
+name: Packaging
+on: pull_request
```

Not fond of abusing the CI to build on every PR :confused: 
Maybe only on stuff that touches the info.xml

```suggestion
on: 
  pull_request:
    paths: 
      - 'appinfo/info.xml'

```

---

while it's very seldom there have been issues with the build process only caught with such a check in the Mail app repo

the purpose is not to create an artifact. it's more of a smoke test. it's run just to make sure nothing throws an error